themes/wilson_theme_starterkit/components/atoms/button/button.twig

```diff
@@ -0,0 +1,3 @@
+{% for item in items %}
```

Comment:
It seems to me like there's a bit more Drupal logic in here than would be strictly necessary, particularly the loop. Thinking for a moment about this storybook component being consumed from another technology, they would have to provide an array structure that feels quite awkward.

I would expect this atom to just be a single anchor tag and to pass the constituent parts in as plain variables.

Something like:

```
<a href="{{ url }}" class="btn {{'btn--' ~ type}} {{ external ? 'is-external'}}" {% external %} target="_blank"{% endif %}>{{ title }}</a>
```

In the Drupal template:

```
{% for item in items %}
  {% include "@storybook/atoms/button/button.twig" with {
    "type": "primary",
    "url": item.content['#url'],
    "title": item.content['#title'],
    "external": item.content['#url'].isExternal,
  } %}
{% endif %}
```

Reply:
Agreed, altered